app/integrations/google_workspace/google_directory.py

```diff
@@ -122,30 +106,16 @@ def list_group_members(group_key):
     Ref: https://developers.google.com/admin-sdk/directory/reference/rest/v1/members/list
     """
 
+    if not delegated_user_email:
+        delegated_user_email = DEFAULT_DELEGATED_ADMIN_EMAIL
     scopes = ["https://www.googleapis.com/auth/admin.directory.group.member.readonly"]
-
-    service = get_google_service(
+    return execute_google_api_call(
         "admin",
         "directory_v1",
-        delegated_user_email=GOOGLE_DELEGATED_ADMIN_EMAIL,
-        scopes=scopes,
+        "members",
+        "list",
+        scopes,
+        delegated_user_email,
+        paginate=True,
+        groupKey=group_key,
```

Comment:
Not sure if you need to specify maxRessults here as not sure what is the default (it might be insufficient or not, just not sure)

Reply:
I can specify the max allowed (200). Doesn't seem required though based on the API ref.